### DISCUSSION TITLE
Any update is coming? (dead project?)

### DISCUSSION THREAD

**Discussion by alekslyse** [kinduff/firefly_iii_exporter#8] [General]
I see its some parts that is not added, on the roadmap, and not been updated for over a year. Has this project been abandoned or is it someone that forked and made more metric exports?

**Comment by kinduff** [comment]
Project is not dead but it works okay as is. Haven't made an update for the rest of the features because I don't use them. If you need one of the remaining metrics, feel free to add a comment in the corresponding issue or contribute with a PR.
